app/models/model.py

```diff
@@ -358,56 +261,48 @@ class ShadeAbridged(BaseModel):
 
     geometry: Face3D
 
-    properties: ShadeProperties
+    boundary_condition: Union[Ground, Outdoors, Adiabatic, Surface]
```

Comment:
The only 2 boundary conditions that are legal for a `Door` are `Outdoors` and `Surface`.  So you should remove `Ground` and `Adiabatic` from here.